editor/index.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
 import { BrowserRouter, Route } from 'react-router-dom';
 import Editor from './Editor';
 import './index.css';
 import 'higlass/dist/hglib.css';
 
-const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement);
-root.render(
+ReactDOM.render(
     <BrowserRouter>
         <Route component={Editor} />
-    </BrowserRouter>
+    </BrowserRouter>,
+    document.getElementById('root') as HTMLElement
```

Comment:
Should this also have to be reverted?

Reply:
It doesn't have to be, but my thinking was to be consistent in the editor with how we expect most will consume Gosling.js package. When we upgrade react (and it does break in the future), the editor will also break.

Reply:
Make sense. I am happy with the current version.